kubernetes/cray-opa/tests/opa/run_tests/run_tests.go

```diff
@@ -180,6 +190,20 @@ func main() {
 	}
 	fmt.Println("user token:", userToken)
 
+	args = createTokenArgs{rrole: "tenant-admin", groups: "vcluster-coke-tenant-admin", issuer: keycloakIssuer, aud: shastaAud, typ: "Bearer"}
```

Comment:
Please again be clear that these are tenant admin tests (e.g., TenantToken vs. TenantAdminToken. Also, please reflect the 'admin' role tests in symbol naming and documentation to be 'infrastructure admin'.